ecosystem/sep-0006.md

```diff
@@ -367,13 +368,38 @@ Name | Type | Description
 `fee_percent` | float | (optional) Percentage fee (if any). In units of percentage points.
 `extra_info` | object | (optional) JSON object with additional information about the deposit process.
 
+`instructions` fields:
+
+An object with SEP-9 financial account fields as keys and its values are objects with the following fields:
+
+Name | Type | Description
+-----|------|------------
+`description` | string | A human-readable description of the field. This can be used to provide any additional information about fields that are not defined in the SEP-9 standard.
+
 `extra_info` fields:
 
 Name | Type | Description
 -----|------|------------
 `message` | string | (optional) Additional details about the deposit process.
 
-Bitcoin response example:
+##### Examples
+
+Bank payment example:
+```json
+{
+  "instructions": {
+    "bank_number": {
+      "description": "Bank number",
+    },
+    "bank_branch_number": {
+      "description": "Bank branch number",
+    }
+  },
+  "id": "9421871e-0623-4356-b7b5-5996da122f3e",
+}
+```
```

Comment:
> Lets update all the examples (anyone using how already knows how to use it, everyone else can benefit from more examples using instructions.

Sure.

> Finally, we're providing information with the `instructions` object, not requesting it. So instead of having SEP-9 fields be objects with a `description`, we need the actual values from the organization:

Oops! Thanks for catching it. I think we should keep the `description` in case SEP-9 fields are not sufficient to describe the deposit instructions.

> I think we're missing SEP-9 fields for crypto addresses and memos in the case of withdraws. Can you add those to SEP-9 before merging this PR? 

Sure.

Reply:
>we're missing SEP-9 fields for crypto addresses and memos in the case of withdraws
Should we also add chain identifier? If user want to receive ERC-20 (e.g stablecoin), it's unclear which chain should be used

Reply:
Lets use SEP-38's asset identification format, which has the `stellar` and `iso4217` schemes already. Other schemes can be added if an anchor wants to offer ethereum -> stellar bridging, for example.

Reply:
To be clear, this would be a withdrawal type listed in the `/info` response and not a SEP-9 field, right?

Reply:
Ah you're right, we already have those fields defined in SEP-6.. never mind on this request then.

Reply:
I think we should still add `crypto_address` and `crypto_memo` to the SEP-9 fields. Chain identifier is handled by the SEP-6 withdraw types.

Reply:
https://github.com/stellar/stellar-protocol/pull/1382